GoogleTestAdapter/Core/Scheduling/TestDurationSerializer.cs

```diff
@@ -163,7 +163,7 @@ private GtaTestDurations LoadTestDurations(string durationsFile)
                 }
                 catch (InvalidOperationException e) when (e.InnerException is XmlSchemaValidationException)
                 {
-                    throw new InvalidTestDurationsException($"Invalid file {durationsFile}. {e.InnerException.Message}", e.InnerException);
+                    throw new InvalidTestDurationsException($"{Resources.InvalidFile} {durationsFile}. {e.InnerException.Message}", e.InnerException);
```

Comment:
Why durationsFile is not a fill-in? Also exception message is a fill-in in many other resource strings.